src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -115,15 +115,32 @@ private static TryCatchStatement composeLroUnaryRpcMethodSampleCode(
   }
 
   private static TryCatchStatement composePagedUnaryRpcMethodSampleCode(
-      Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary paged rpc method.
-    // TODO(summerji): Add unit tests.
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add unit test.
+    // Assign each method arguments with default value.
+    List<Statement> bodyStatements =
+        arguments.stream()
+            .map(
+                methodArg ->
+                    ExprStatement.withExpr(
+                        assignMethodArgumentWithDefaultValue(methodArg, resourceNames)))
+            .collect(Collectors.toList());
+    // For loop client on iterateAll method.
+    // e.g. for (LoggingServiceV2Client loggingServiceV2Client :
+    // loggingServiceV2Client.ListLogs(parent).iterateAll()) {
```

Comment:
Nit: Can we format this better, with indenting and the ending brace on a separate line?

Reply:
Sure.